practicalai/practical-ai-27.md

```diff
@@ -72,9 +72,9 @@ In the prospective studies not only is the individual first informed what it is
 
 We can think of this in two parallel avenues. One is the typical AI, big data signs approach. We try to create features of all colors and shapes, and throw them against the wall and see what sticks. But at the same time, of course the space of features is, for all practical purposes, infinite, so you always need knowledge. So at the same time, what we are doing is by interacting with clinicians and medical researchers, we are trying to open up their minds and trying to understand how the features and the symptoms that they have found to be most relevant can be turned into algorithms. I can give you a very concrete example of both.
 
-In the first case, when we create features, we have results showing that we can discriminate parkinsonian patients that are on the medication \[unintelligible 00:16:34.23\] or off the medication, using features that include frequency components of the voice that are not detectable by the human ear, but they are still there because the drug is psychoactive, so it affects your network system, and of course, trivially, it affects your voice.
+In the first case, when we create features, we have results showing that we can discriminate \[between\] parkinsonian patients that are on the medication (Levodopa) or off the medication, using features that include frequency components of the voice that are not detectable by the human ear, but they are still there because the drug is psychoactive, so it affects your network system, and of course, trivially, it affects your voice.
```

Comment:
I haven't seen an example of corrected context like the `\[between\]` you've added, but I can see an argument for it in this case where discrimination means something else without it. I'll tag @jerodsanto to make sure it's kosher.

Reply:
I dig it 👍